app/Commands/Dev/Tree/Compile/Options.hs

```diff
@@ -0,0 +1,23 @@
+module Commands.Dev.Tree.Compile.Options
+  ( module Commands.Dev.Tree.Compile.Options,
+    module Commands.Extra.Compile.Options,
+  )
+where
+
+import Commands.Extra.Compile.Options
+import CommonOptions
+import Data.List.NonEmpty qualified as NonEmpty
+
+treeSupportedTargets :: NonEmpty CompileTarget
+treeSupportedTargets =
+  NonEmpty.fromList
```

Comment:
we have `nonEmpty'` in the prelude to avoid having to import `Data.List.NonEmpty` that often

Reply:
done